### DISCUSSION TITLE
Win10 iSCSI target install hangs after selecting iSCSI drive to install to

### DISCUSSION THREAD

**Discussion by skysurf76** [ipxe/ipxe#1315] [Q&A]
I am booting a diskless PC via a thumb drive with iPXE on it. I sanhook to iSCSI target (win server 2022), run wimboot from Apache running on the win server, after WinPE loads I start Win10 install from setup files on a network share.  The iSCSI drive shows up as unallocated space, I select it and hit next and I just get an hourglass after that.

I started with 100gig iSCSI target, then I tried a 30gig target and I still just get the hourglass.  I'm using the built in iSCSI target server on the Win server.

I'm so close.  Can anyone offer advice?

**Comment by psychrabi** [comment]
The issue most likely write permission to the iscsi target. Try adding the MAC address of the client to the Allowed IQN
